packages/fast-components-react-base/src/horizontal-overflow/horizontal-overflow.tsx

```diff
@@ -25,6 +28,11 @@ export interface HorizontalOverflowState {
     itemsHeight: number;
 }
 
+export interface ItemsWidthAndTotalWidth {
```

Comment:
As per my comment below, I think this is unnecessary and we should not need to have an interface if we just split the method for getting the item widths and available width.